source/manual/get-started.html.md

```diff
@@ -242,6 +242,22 @@ After your PR has been merged, someone from the `govuk-administrators`
 or `govuk-internal-administrators` group needs to deploy the
 `infra-security` project.
 
+#### New access controls
```

Comment:
I think having a "New access controls" section will age poorly... It's not going to be new forever!

I'd prefer to update the existing docs just to say how to access the new roles, and leave out the contextual bit around removing the RE out-of-hours rota.

Reply:
Sounds good 👍

Reply:
Done